<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新增下载任务</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>81</width>
     <height>241</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>391</width>
     <height>231</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>36</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>地图</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_mapTypes">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>15</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>70</y>
      <width>36</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>级别</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_level">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>70</y>
      <width>49</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>19</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>36</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>范围</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_range">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>121</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>70</y>
      <width>72</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>-------&gt;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_level_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>49</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>19</number>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>超时</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_timeout">
       <property name="text">
        <string>30</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>秒</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>重试次数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_retry">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
